src/main/java/com/sparkplatform/api/core/Response.java

```diff
@@ -43,6 +43,10 @@
 	private JsonNode rootNode;
 	private SparkAPIClientException exception;
 	
+	public Response() {
+		super();
+	}
```

Comment:
Basically, all of [these calls](https://github.com/sparkapi/SparkJava/blob/master/src/test/java/com/sparkplatform/api/core/ConnectionApacheHttpTest.java#L56) were erroring out [here](https://github.com/sparkapi/SparkJava/blob/master/src/main/java/com/sparkplatform/api/core/Response.java#L49), as that was the constructor being called, and `.getMessage()` was being sent to `null`.

I just added in a default constructor, since we're expecting `Response` to be constructed with nothing passed to the constructor.